data/lib/core/storages.lua

```diff
@@ -0,0 +1,12 @@
+Storage = {
+    annihilator = {
+        Reward = 30015,
+    }
```

Comment:
is there anything else that would go into `annihilator`? Why so much nesting?

Reply:
in my opinion because it is a quest it should be separated from the others, in case it is necessary to add more keys in the future.